versions.tf

```diff
@@ -4,23 +4,11 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.56"
+      version = ">= 3.64"
     }
-    local = {
-      source  = "hashicorp/local"
-      version = ">= 1.4"
-    }
-    kubernetes = {
-      source  = "hashicorp/kubernetes"
-      version = ">= 1.11.1"
-    }
-    cloudinit = {
-      source  = "hashicorp/cloudinit"
-      version = ">= 2.0"
-    }
-    http = {
-      source  = "terraform-aws-modules/http"
```

Comment:
Do you know if there is still a need for this `http` provider we forked some time ago?

Reply:
No I don't believe we need this anymore - but might be best to just archive it in case others are actually using it even if the EKS module is not